compiler/ballerina-parser/src/main/resources/token_en_LK.properties

```diff
@@ -49,6 +49,7 @@ const = const
 final = final
 typeof = typeof
 is = is
+!is = !is
```

Comment:
This is not enough right? we need to update the logic in https://github.com/ballerina-platform/ballerina-lang/blob/6bbce7083c293a84ed7c20878a8efe55a7602504/compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaLexer.java#L1309-L1312

Reply:
Updated c4f49866c788740fa8ae9fb64dd9e97ffcf15b59